code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -173,13 +182,16 @@
 /obj/machinery/atmospherics/components/unary/cryo_cell/nap_violation(mob/violator)
 	open_machine()
 
-/obj/machinery/atmospherics/components/unary/cryo_cell/process(delta_time)
-	if(!on)
-		return
-
-	if(!is_operational())
+/obj/machinery/atmospherics/components/unary/cryo_cell/on_set_is_operational(old_value)
+	if(old_value) //Turned off
 		on = FALSE
+		end_processing()
```

Comment:
I had actually planned for this myself, disabling processing when not in use

Reply:
If there is other machines you think I should look at for this kind of change, I can see about adding it.